JenkinsfileRT_dev

```diff
@@ -104,7 +104,7 @@ bc1.build_cmds = [
 ]
 bc1.build_cmds = PipInject(env.OVERRIDE_REQUIREMENTS) + bc1.build_cmds
 bc1.test_cmds = [
-    "pytest --cov-report=xml --cov=./ -r sxf -n auto --bigdata --slow \
+    "pytest --cov-report=term-missing --cov=./ -r sxf -n auto --bigdata --slow \
```

Comment:
Why run coverage here? I don't think we are uploading or using it in this case and coverage just dramatically slows down running python programs in general

Reply:
good point, we can get rid of it there